etc/config/gimple.defaults.properties

```diff
@@ -0,0 +1,64 @@
+# Default settings for GIMPLE
+compilers=&gcc
+defaultCompiler=gimplegdefault
+demangler=c++filt
+objdumper=objdump
+postProcess=
+supportsBinary=true
+supportsBinaryObject=true
+binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*)$
+stubRe=\bmain\b
+stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
+supportsLibraryCodeFilter=true
+
+group.gcc.compilers=gimpleg44:gimpleg45:gimpleg46:gimpleg47:gimpleg48:gimpleg5:gimpleg6x:gimpleg7:gimpleg8:gimpleg9:gimpleg10:gimpleg11:gimplegdefault
```

Comment:
judging by the ChangeLog, -fgimple is only available since gcc 9 (see https://gcc.gnu.org/cgit/gcc/commit/?id=1ee62b926411dda2bffbca8019b494481183ee15)